infoschema/metric_schema.go

```diff
@@ -925,4 +925,1055 @@ var MetricTableMap = map[string]MetricTableDef{
 		PromQL: `pd_region_syncer_status{$LABEL_CONDITIONS}`,
 		Labels: []string{"instance", "type"},
 	},
+	"tikv_engine_size": {
+		PromQL:  `sum(tikv_engine_size_bytes{$LABEL_CONDITIONS}) by (instance, type)`,
+		Labels:  []string{"instance", "type"},
```

Comment:
The `tikv_engine_size_bytes` has a `db` label which indicates column family.

Reply:
Great catch, The `db` label doesn't contain in the Grafana, Then I doesn't know this label, I will add it soon.